ledger/store/src/transaction/deployment.rs

```diff
@@ -257,7 +283,7 @@ pub trait DeploymentStorage<N: Network>: Clone + Send + Sync {
         })
     }
 
-    /// Returns the transaction ID that contains the given `program ID`.
+    /// Returns the latest transaction ID that contains the given `program ID`.
     fn find_transaction_id_from_program_id(&self, program_id: &ProgramID<N>) -> Result<Option<N::TransactionID>> {
```

Comment:
It's already quite verbose, but we may want to consider renaming this to `find_latest_transaction_id_from_program_id`.